README.md

```diff
@@ -16,18 +16,31 @@ version of Firefox.
 Installation
 ------------
 
-OpenWPM has been developed and tested on Ubuntu 14.04/16.04. An installation
-script, `install.sh` is included to install both the system and python
-dependencies automatically. A few of the python dependencies require specific
-versions, so you should install the dependencies in a virtual environment if
-you're installing a shared machine. If you plan to develop OpenWPM's
-instrumentation extension or run tests you will also need to install the
-development dependencies included in `install-dev.sh`.
-
-It is likely that OpenWPM will work on platforms other than Ubuntu, however
```

Comment:
This comment should remain since this PR does not add full support for Mac and Windows.